.gitignore

```diff
@@ -14,6 +14,9 @@ cpp/unreal_projects/SpearSim/Content/MSPresets
 cpp/unreal_projects/SpearSim/Content/Spear/Scenes/warehouse_0000
 cpp/unreal_projects/SpearSim/Content/Stanford
 cpp/unreal_projects/SpearSim/Content/StarterContent
+cpp/unreal_projects/SpearSim/Content/ThirdPerson
```

Comment:
Add these folders and any map(s) you care about to `python/spear/tools.py` because this is a requirement for building paks correctly in UE 5.5.

Reply:
Do we need to add ignore statements to account for the new examples?